components/retro-go/targets/rachel-esp32/config.h

```diff
@@ -0,0 +1,101 @@
+    ILI9341_CMD(0x3A, 0x55);                                                                                                    \
+    ILI9341_CMD(0x20);                                                                                                          \
+    ILI9341_CMD(0xCF, 0x00, 0xc3, 0x30);                                                                                        \
+    ILI9341_CMD(0xED, 0x64, 0x03, 0x12, 0x81);                                                                                  \
+    ILI9341_CMD(0xE8, 0x85, 0x00, 0x78);                                                                                        \
+    ILI9341_CMD(0xCB, 0x39, 0x2c, 0x00, 0x34, 0x02);                                                                            \
+    ILI9341_CMD(0xF7, 0x20);                                                                                                    \
+    ILI9341_CMD(0xEA, 0x00, 0x00);                                                                                              \
+    ILI9341_CMD(0xC0, 0x1B);                                                                                                    \
+    ILI9341_CMD(0xC1, 0x12);                                                                                                    \
+    ILI9341_CMD(0xC5, 0x32, 0x3C);                                                                                              \
+    ILI9341_CMD(0xC7, 0x91);                                                                                                    \
+    ILI9341_CMD(0x36, (0xA0| 0X08));                          /* 1.3 inch 0XA0|0X08  (2.4 inch  0x00|0x80) */                   \
+    ILI9341_CMD(0xB1, 0x00, 0x10);                                                                                              \
+    ILI9341_CMD(0xB6, 0x0A, 0xA2);                                                                                              \
+    ILI9341_CMD(0xF6, 0x01, 0x30);                                                                                              \
+    ILI9341_CMD(0xF2, 0x00);                                                                                                    \
+    ILI9341_CMD(0x26, 0x01);                                                                                                    \
+    ILI9341_CMD(0xE0, 0xD0, 0x00, 0x05, 0x0E, 0x15, 0x0D, 0x37, 0x43, 0x47, 0x09, 0x15, 0x12, 0x16, 0x19);                      \
+    ILI9341_CMD(0xE1, 0xD0, 0x00, 0x05, 0x0D, 0x0C, 0x06, 0x2D, 0x44, 0x40, 0x0E, 0x1C, 0x18, 0x16, 0x19);                      \
+    ILI9341_CMD(0x11);                                                                                                          \
+    ILI9341_CMD(0x29);     
+
+// Input
+// Refer to rg_input.h to see all available RG_KEY_* and RG_GAMEPAD_*_MAP types
+#define RG_GAMEPAD_ADC_MAP {\
+}
```

Comment:
You should comment those lines  so that the ADC driver isn't built, since it's not needed by your device.